source/sass/type.scss

```diff
@@ -41,7 +41,7 @@ a {
   line-height: 1.5;
 }
 
-@mixin yellow-underline-nav-link($active-class-name, $font-family: "Zilla Slab") {
+@mixin yellow-underline-nav-link($active-class-name, $font-family: "Nunito Sans") {
```

Comment:
@taisdesouzalessa 
This code change will make **all** multi-page nav links use Nunito Sans (which includes the links on the opportunity page). Sounds good? If not, I revert opportunity page links back to Zilla Slab.

Reply:
That sounds good, Mavis. Since we will be using this style across pages, could we use Nunito Sans Semi Bold instead or the regular weight? So the element on the sidebar looks more like a menu and less like a body text. For all nav items - desktop and mobile. I'll alter the mockups to accommodate that change. Sounds good?
<img width="925" alt="image" src="https://user-images.githubusercontent.com/22157921/35646258-6b83f2e6-0683-11e8-82a8-a80c14ff71e5.png">

Reply:
Let's discuss this in a new ticket. From the [style guide](https://foundation.mozilla.org/style-guide/) it seems like all the anchor styles are in bold (`font-weight: 700`). I hesitate to throw an exception there (at least not in this PR).

Will create a ticket and tag you @taisdesouzalessa and @gvn